MSP432/DistanceSensor/UltrasonicSensorCommon.c

```diff
@@ -92,7 +92,12 @@ static UltrasonicSensorCommon_t instance;
 
 enum
 {
-    PeriodToDoUltrasonicCycleMs = 10
+    PeriodToDoUltrasonicCycleMs = 10,
+};
+
+enum UltraSonicDistanceCorrection
+{
+    DistanceCorrectionCm = 5,
```

Comment:
Can add this to the above enum. Also, no need to give it a name.

Reply:
Done . Ready to merge changes.